src/index.js

```diff
@@ -70,15 +70,16 @@ const VueSmoothScroll = {
 
         const hash = vnode.data.attrs.href;
 
+        let resolvedConfig = Object.assign({}, defaultValue);
```

Comment:
@Yuliang-Lee I hope this implementation works as you expected?

Reply:
yes, it's ok